### ISSUE TITLE
Feature request: scroll active menu item into view

### ISSUE THREAD

**Issue by 5tm** [alex-shpak/hugo-book#485]
Hugo-book is a great theme, thanks for your hard-working @alex-shpak. Could you do me a favor. I want to make the active menu item auto scroll into view. I am not a programmer and I am not good at programming. After google, I got these script.
`function scroll(){`
 ` let e = document.querySelector( '.book-menu a.active' );`
`    e.scrollIntoView({ behavior: 'smooth', block: 'center', inline: 'start' })`
`}`
`setTimeout(scroll, 0);`
It worked, but it created more problem. 

I like your no jquery idea, in my opinion javascript is convenient for developers but not for visitors, less javascript is more.
I think this can be a useful feature. can you improve it?

**Comment by alex-shpak** [comment]
Hi!
You trying to put an active page (selected page) in left menu on screen when loading? (trying to understand a usecase)

**Comment by alexatadi** [comment]
Agree with this feature, also hope the TOC (say head #2 and #3 in the right panel) can synchronized scrolling and highlighting follows the contents in viewing.

**Comment by alexatadi** [comment]
found the answers:  #235

**Comment by 5tm** [comment]
> Hi! You trying to put an active page (selected page) in left menu on screen when loading? (trying to understand a usecase)

Sorry, i missed your message. the usecase is making the hugo-book more like a real book. 

I made the menu always fixed in mobile view(cause it works well in pc). I clicked a link(maybe chapter 23) and start to read. 2 minutes later I felt its boring. I clicked the menu and want to know which chapter i am. then i jump to another chapter(maybe chapter 25). I know this is a bad reading behavior. But this is the way i am reading a real book.

If you think this can be defined as a usecase, maybe make the site title and language option fixed(can't be scrolled) is better. 

sorry for my poor English expression.
